src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -201,4 +207,19 @@ protected boolean isConsideredNull(Object propertyValue) {
         }
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
+
+    @Override
+    public void parseComplexValues(Object e, Map<String, Value> values) {
+        parseValue(e, Value.of(values));
```

Comment:
Shouldn't we reather implement the logik here?

Reply:
Parsing is done here how